Documentation/raspi-spi.md

```diff
@@ -99,6 +99,16 @@ dtoverlay=spi0-1cs,cs0_pin=27,no_miso
 
 There is only for SPI0 that you can use, in both cases with 1 or 2 Chip Select pin the `no_miso`option.
 
+**Important note**: Those overlays are only supported in the very last Raspberry Pi OS. You will get the `System.IO.IOException: Error 2. Can not open SPI device file '/dev/spidev0.0'` error message if you are using them on an older version. You can use `spi0-cs` where in the previous examples you had `spi0-2cs` or `spi0-1cs`. 
```

Comment:
Since you're changing this I'd recommend to also use `sudo raspi-config nonint do_spi 0` on raspbian since it's easier

Reply:
Added.